docs/tutorials/Manual-Signature-Verification/Readme.md

```diff
@@ -0,0 +1,268 @@
+# Verifying PowerAuth Signatures On The Server
+
+<!-- AUTHOR joshis_tweets 2020-06-04T00:00:00Z -->
+<!-- SIDEBAR _Sidebar.md sticky -->
+
+In this tutorial, we will show you how to verify PowerAuth signatures manually on the server-side. While the task is relatively simple, it is very sensitive to any minor inconsistencies. Do not get frustrated if the signature verification does not work for you the first time. If you get stuck, do not hesitate to contact our engineers for help.
```

Comment:
Perhaps we should add a link to contact page?

Reply:
I will have a look - it is not easy to build a link to the dev portal support section... I will give it a shot...